grpc-api/src/main/java/io/deephaven/grpc_api/barrage/BarrageStreamGenerator.java

```diff
@@ -304,10 +313,13 @@ private InputStream getInputStream(final SubView view) throws IOException {
             nodeOffsets.setSize(0);
             bufferInfos.setSize(0);
 
+            final MutableInt bufferOffset = new MutableInt();
             final ChunkInputStreamGenerator.FieldNodeListener fieldNodeListener =
                     (numElements, nullCount) -> nodeOffsets.add(BarrageFieldNode.createBarrageFieldNode(builder, numElements, nullCount, 0, 0));
-            final ChunkInputStreamGenerator.BufferListener bufferListener =
-                    (offset, length) -> bufferInfos.add(new ChunkInputStreamGenerator.BufferInfo(offset, length));
+            final ChunkInputStreamGenerator.BufferListener bufferListener = (offset, length) -> {
```

Comment:
Why does `BufferListener` still take an offset, when we always send `0` and use a `MutableInt` pattern to accumulate the offsets?